src/AddressRegistry.Api.BackOffice.Abstractions/Requests/AddressCorrectRejectionRequest.cs

```diff
@@ -10,7 +10,7 @@ namespace AddressRegistry.Api.BackOffice.Abstractions.Requests
     using StreetName.Commands;
 
     [DataContract(Name = "CorrigerenAfkeuringAdres", Namespace = "")]
-    public class AddressCorrectRejectionRequest : AddressBackOfficeCorrectAddressFromRejectedRequest, IRequest<ETagResponse>
+    public class AddressCorrectRejectionRequest : BackOfficeCorrectAddressFromRejectedRequest, IRequest<ETagResponse>
```

Comment:
Shouldn't it be CorrectAddressRejection
like used here https://github.com/Informatievlaanderen/address-registry/blob/3f5ee090c4f6b342408395968d9a7ce120f116e3/src/AddressRegistry.Api.BackOffice.Handlers.Sqs/Handlers/CorrectRejectionSqsHandler.cs#L12 ?